app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+my_dict = {"lucky_number": lucky_number, "pi": pi, "one_is_a_prime_number": one_is_a_prime_number, "name": name, "my_favourite_films": my_favourite_films, "profile_info": profile_info, "marks": marks, "collection_of_coins": collection_of_coins }
+sorted_variables = {}
+
+for key, value in my_dict.items():
+    if isinstance(value, (list, set, dict)):
+        sorted_variables["mutable"] = []
+    else:
+        sorted_variables["immutable"] = []
+
+for key, value in my_dict.items():
+    if isinstance(value, (list, set, dict)):
+        sorted_variables["mutable"] += [key]
+    else:
+        sorted_variables["immutable"] += [key]
```

Comment:
No additional logic is required - just pass the variables to the correct list in the dictionary.

Reply:
hello! 
the solution has been changed and pushed
please check new one